lib/src/logic.dart

```diff
@@ -846,4 +846,21 @@ class Logic {
       getRange(0, startIndex),
     ].swizzle();
   }
+
+  /// Returns `1` (of [width]=1) if the [Logic] calling this function is in
+  /// [List] list. Else `0` (of [width]=1) if not present.
```

Comment:
Instead of `[List] list` we should put just `[list]` here

In places where you're referring to `the [List]` in general, we can just say `[list]`